### PR TITLE
Update readme and startup.sh script

### PR DESCRIPTION
**Pull request by edwardkerry**
Updates README in favour of GOVUK Docker and limits startup script to only operate in live mode.

[trello](https://trello.com/c/Psf7EzPB/204-move-away-from-startupsh)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Assisted digital satisfaction surveys:
 
 ## Technical documentation
 
-Frontend is a Ruby on Rails application that renders the citizen-facing part of formats stored in the Content Store. It looks up the passed-in slug in the Content Store.
+Frontend is a Ruby on Rails application and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html). It renders the citizen-facing part of formats stored in the Content Store. It looks up the passed-in slug in the Content Store.
```
Comment (ChrisBAshton) [inline README.md:84]: I think the "It renders the..." part onwards can be moved to the top of the README. We should also remove the HTML at the top of the README (part of this work is standardising the README structure, and that HTML is not standard!)

File: README.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ Assisted digital satisfaction surveys:
...
 
 It also serves the homepage as a hard-coded route.
```
Comment (ChrisBAshton) [inline README.md:86]: This part can also be moved to the intro.

The "Calendar JSON" section below feels like something that would be better moved into its own document in the `docs/` folder, which we can link to from a "Further documentation" subsection that we've been adding to READMEs.

Finally, the `Additional information for calendars` sections onwards could also have the same treatment applied. (Sorry I can't comment on the actual lines - GitHub seems to have removed a feature!)

**Review by ChrisBAshton — COMMENTED** [review]
Looking good, Ed! I just have a few comments below:

File: README.md (hunk 1 above)
Comment (edwardkerry) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (edwardkerry) [reply README.md]: 👍

File: docs/calendars.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+
```
Comment (ChrisBAshton) [inline docs/calendars.md:1]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-<h1>GOV.UK Mainstream, Home & Search Frontend</h1>
-
-<p>Application to serve some mainstream formats and the homepage for GOV.UK.</p>
+# Frontend
 
+Frontend renders the citizen-facing part of formats stored in the Content Store, and
+some hard-coded routes including the GOV.UK homepage.
```
Comment (ChrisBAshton) [inline README.md:4]:
```suggestion
some hard-coded routes including the GOV.UK homepage.

```

File: README.md [hunk 5]
```diff
@@ -118,9 +91,11 @@ The division `title` attribute is optional.  If this is not present the slug wil
 
 ### Running the application
 
-If you are [using docker](https://github.com/alphagov/govuk-docker) to run the application (which is advised) it will be available on the host at http://frontend.dev.gov.uk/
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
+
+If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
 
-To run the application standalone, run [static](https://github.com/alphagov/static) and execute the following command:
+If you are using the `startup.sh` script, first run [static](https://github.com/alphagov/static) and execute the following command:
```
Comment (ChrisBAshton) [inline README.md:98]: This section below needs updating, as Static no longer lives on localhost:3013. It is [now started with govuk-docker](https://github.com/alphagov/static#technical-documentation), so we'd have to set `PLEK_SERVICE_STATIC_URI` to http://static.dev.gov.uk.

File: startup.sh [hunk 6]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-bundle install
+bundle check || bundle install
```
Comment (ChrisBAshton) [inline startup.sh:3]: Nice! Though is this change necessary? I don't think I've seen this pattern used elsewhere, and from memory, running `bundle install` when I have all the local dependencies already doesn't re-download/re-install the gems, it just exists quite quickly 🤔

**Review by ChrisBAshton — APPROVED** [review]
Looking good! A couple of syntax tweaks below, but ready to go after that. I've approved the PR ahead of time. 👍

File: startup.sh (hunk 6 above)
Comment (edwardkerry) [reply startup.sh:3]: I borrowed it from here
https://github.com/alphagov/info-frontend/pull/891/commits/fe012d5e8120dab6145459fea888af2f6ee5325b
I believe that `check` is quicker than `install`, but it is probably a marginal gain for apps without a huge amount of dependencies. Happy for to remain or be removed!

File: README.md [hunk 7]
```diff
@@ -118,49 +92,28 @@ The division `title` attribute is optional.  If this is not present the slug wil
 
 ### Running the application
 
-If you are [using docker](https://github.com/alphagov/govuk-docker) to run the application (which is advised) it will be available on the host at http://frontend.dev.gov.uk/
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
 
-To run the application standalone, run [static](https://github.com/alphagov/static) and execute the following command:
+If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
+
+If you are using the `startup.sh` script, first run [static]()https://github.com/alphagov/static) and execute the following command:
```
Comment (benthorner) [inline README.md:99]: Just a fly-by comment in case you want to fix: this is incorrect, since `./startup.sh --live` will point at the live instance of Static, so you don't need it running locally (or to set the corresponding environment variable).
